test/ui-test.ts

```diff
@@ -24,8 +25,9 @@ hosts.forEach(function (host) {
             assert.equal(serverResponse["status"], 200);
 
             // Call startDebugging to start dev-server and sideload
-            const devServerCmd = `npm run dev-server -- --config ./test/webpack.config.js`;
-            await startDebugging(manifestPath, AppType.Desktop, toOfficeApp(host), undefined, undefined, devServerCmd);
+            const devServerCmd = `npm run dev-server -- --config ./test/webpack.config.js `;
+            const devServerPort = parseNumber(process.env.npm_package_config_dev_server_port || 3000);
+            await startDebugging(manifestPath, AppType.Desktop, toOfficeApp(host), undefined, undefined, devServerCmd, devServerPort);
```

Comment:
are there changes needed for stopDebugging to not call "npm run unload"?

Reply:
(I can't tell from the diff view...)

Reply:
I found how to expand it. It looks OK.

Reply:
I think this is correct - this is just a straight merge of Angular Ts